semanticdb/semanticdb3/semanticdb3.proto

```diff
@@ -36,13 +41,11 @@ message Location {
 
 message Type {
   enum Tag {
+    // reserved 2, 3, 4, 5;
```

Comment:
@thesamet Is it expected that this line fails? https://github.com/google/protobuf/issues/3458 suggests that Protocol Buffers have support for `reserved` in enums.

Reply:
@xeno-by that depends on the version of protoc you are using. As far as I can tell protoc 3.5.x supports `reserved`. It looks like your build is using a custom protoc (https://github.com/scalameta/scalameta/blob/7a4f642c6173598f7edd8ceb4398af906f927d12/build.sbt#L543) I'm not using which version it is.

Reply:
This part of build configuration only uses custom protoc on `NixOS`, otherwise it falls back to the standard `(PB.runProtoc in Compile).value`. How do I find what version of `protoc` is bundled with ScalaPB?

Reply:
The default behavior is to call into protoc-jar and passing a version to it. It can be controlled/examined using [`PB.protocVersion`](https://github.com/thesamet/sbt-protoc/blob/c29d4d6c6074b25edda1efdabd89eddea27fe770/src/main/scala/sbtprotoc/ProtocPlugin.scala#L66). The command invoked is also printed so you might be able to see the version there as well. Can you link/paste the full output of whatever is failing?

Reply:
```
> semanticdb3JVM/compile
[info] Compiling 1 protobuf files to /Users/eburmako/Projects/scalameta/semanticdb/semanticdb3/.jvm/target/scala-2.12/src_managed/main
[info] Compiling schema /Users/eburmako/Projects/scalameta/semanticdb/semanticdb3/semanticdb3.proto
protoc-jar: protoc version: 340, detected platform: mac os x/x86_64
protoc-jar: executing: [/var/folders/pg/v06175sd2_qb9jbg7c28xr040000gn/T/protocjar5934144415890866447/bin/protoc.exe, --plugin=protoc-gen-scala=/var/folders/pg/v06175sd2_qb9jbg7c28xr040000gn/T/protocbridge4997102442195402814, --scala_out=flat_package,grpc:/Users/eburmako/Projects/scalameta/semanticdb/semanticdb3/.jvm/target/scala-2.12/src_managed/main, -I/Users/eburmako/Projects/scalameta/semanticdb/semanticdb3, -I/Users/eburmako/Projects/scalameta/semanticdb/semanticdb3/.jvm/target/protobuf_external, /Users/eburmako/Projects/scalameta/semanticdb/semanticdb3/semanticdb3.proto]
semanticdb3.proto:44:14: Missing numeric value for enum constant.
[trace] Stack trace suppressed: run last semanticdb3JVM/compile:protocGenerate for the full output.
[error] (semanticdb3JVM/compile:protocGenerate) protoc returned exit code: 1
[error] Total time: 1 s, completed Feb 14, 2018 8:59:13 AM
```

https://github.com/xeno-by/scalameta/commit/142aa521aa6ed295836400a5448eede59e949ab3